openedx_external_enrollments/tests/external_enrollments/tests_pathstream_external_enrollments.py

```diff
@@ -0,0 +1,129 @@
+"""Tests PathstreamExternalEnrollment class file"""
+import logging
+
+from django.test import TestCase
+from mock import Mock, patch
+from opaque_keys.edx.keys import CourseKey
+from testfixtures import LogCapture
+
+from openedx_external_enrollments.external_enrollments.pathstream_external_enrollment import (
+    PathstreamExternalEnrollment,
+)
+
+
+class PathstreamExternalEnrollmentTest(TestCase):
```

Comment:
I'm assuming you expect not to fail when "_post_enrollment" is called without "data" if so, create a test case for that scenario. @Jacatove

Reply:
@Squirrel18 Great observation. I added a try except into  "_post_enrollment" to catch those scenarios caused by an empty/wrong "data"